.github/workflows/clang-format.yml

```
@@ -0,0 +1,34 @@
+name: clang-format-check
+
+on:
+  pull_request:
+    branches:
+    - sycl
+    types: [open, edit, reopen, synchronize]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # checkout PR head
+        ref: '${{github.event.pull_request.head.sha}}'
+    - name: Fetch target branch
+      run: git fetch --no-tags --prune --depth=1 origin +refs/heads/${{github.base_ref}}:refs/remotes/origin/${{github.base_ref}}
+
+    - name: Get clang-format first
+      run: sudo apt-get install clang-format-9
```

I just tried to execute this line locally, and it asks for permission to install the packages. Not sure if GitHub Actions correctly handles this, but you can add `-yqq` to proceed without any quesions from apt.

```suggestion
      run: sudo apt-get install -yqq clang-format-9
```

---

GitHub actions handles this command correctly. I tested it in my private fork.